Makefile

```diff
@@ -8,7 +8,7 @@ endif
 TARGETS=awesome
 BUILDDIR=build
 
-all: $(TARGETS) ;
+all: check-unit $(TARGETS) ;
```

Comment:
I don't like this part, sorry. The top-level Makefile is intended for convenience. Using CMake directly should not have different results.

However, I'll let this be.